### ISSUE TITLE
@param annotation not inherited for renamed parameters in overridden methods

### ISSUE THREAD

**Issue by julienfalque** [phpstan/phpstan-strict-rules#78]
When a child class overrides a method and renames one if its parameter, the `@param` annotation of the parameter on the parent method is ignored. I think it should still be inherited.

In the following example:

```php

class Foo
{
    /**
     * @param string $foo
     */
    public function test($foo) { }
}

class Bar extends Foo
{
    public function test($bar) { }
}
```

PHPStan will report the following error:
```
Method Bar::test() has parameter $bar with no typehint specified.
```

If I rename `$bar` to `$foo`, the error isn't reported anymore.

**Comment by ondrejmirtes** [comment]
Hi, this issue is known (#671) but unless you want to send a PR for this yourself, I consider it a wontfix.
